### ISSUE TITLE
Update experience prototype and visual design wrap up

### ISSUE THREAD

**Issue by austinhernandez** [18F/cms-hitech-apd#1748]
### Important links:

- [Invision prototype here](https://gsa.invisionapp.com/share/AKT8E97G36C#/376580993_APD_builder_home)
- [Sketch file on Drive here (Aug 3)](https://drive.google.com/open?id=1AQuiu39d4SUVOxKIdH9Yo6qR9u2t9d1i)
- [Mural of process here](https://app.mural.co/t/gsa6/m/gsa6/1563383330690/2e1e8324d51c7631686f6a4c27ccec28752d5ebb)

### Status on design
 - The InVision app is a little buggy. I ran out of time at the end, but it should be in a place where you can put it in front of users.

### Slack convo about next steps on update experience:

**1. Where do you see the short summary fit into the experience if the user selects "Clarify more changes?"**
- Clarify more changes is just the “oops I meant that there are more changes in scope” button that takes u to the edit page. I would imagine that information was still needed on the next page so the same content would be there. I wanted to give an option of in between (in case there was one) of lightweight and full edit. 
 - We would need an area where the state does provide an update. The existing set of fields doesn't necessarily give me (as a reviewer) a sense of what changed.

**2. Would be nice to talk through what you were thinking for future federal fiscal years.**
- The only thing I was anticipating that would change was FY20++ showing up and additional fields being displayed besides the current year. I can talk thru if it’s not the case
 - Makes sense - in that case, the app would have to know to only offer the full edit experience for the APD at that point.

**3.  Where would the user have the option to add additional activities (since the current flow shows only existing activities)?**
The end after all of the activities were looked thru (or skipped). Today I’m making that “do u have a new activity?” screen
 - :thumbsup:

**4. After walking through this, we probably need to document a little more about the close out experience (when an activity is done). But don't want to go down a rabbit hole more than I already have**
Sounds good. I can make a blank placeholder screen. Right now it’s just did you complete it, what was the actual cost, onto the next activity...
 - I think what you have should be fine for now - maybe just have a placeholder text in the current box that notes we need to work through what data we would need to collect for an completed activity.

**5. What happens after the user completes the activity walkthrough? Does the executive summary + budget tables + etc. automatically generate to create the final IAPD-U?**
- After you have finished all the program activities (incl. if you need to add one) complete. The budget tables and everything would update the same way it does the first time you fill out the APD. Which is the results of previous activities and a display of the info filled out in previous pages, budget table and exec summary
 - :thumbsup:

### Wishlist:
1. Don't start the update experience until the eAPD is in a good place.
 - There are many corners we cut to get this as the first robust prototype in front of users. Take their feedback into account before pursuing the next phase of work
2. Take a step back and focus on the product and experience, not having an online version of the Word Doc.
3. Focus on "What are real constraints?" and not "what's been done in the past. Beyond the eAPD software, what can we do to improve the process?
4. If anything, I hope this update experience prototype shows how users are confused by the core of the eAPD. I trust the feedback you get back will help build a sturdier foundation. 

### Fears:
1. We are turning a paper form into an online form with no true product rethinking. 
  - The form is too long and there is a lot of interaction work I wanted to do to improve the UI.
2. We focus too much on future work, that the core product is buggy, unreliable, and unstable.
3. The work we did on improving the current APD gets lost.

**Comment by lauraponce** [comment]
Link to https://github.com/18F/cms-hitech-apd/issues/1692

**Comment by lauraponce** [comment]
Link to https://github.com/18F/cms-hitech-apd/issues/1711

**Comment by lauraponce** [comment]
Saved out the wishlist and fears to bring up at upcoming planning meetings and linked to the update experience issue for testing the prototype, closing this one out!
